### PR TITLE
Contact me  form

### PR DESCRIPTION
**Pull request by GreetyCr**
# Features

- Responsive values
- FormSpree as a way to send emails via form
- Pair programming 

# Collaborator

@JavierAybar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -343,5 +343,22 @@ <h2 class="about-myself__text--title">
...
+      Contact me
+    </h2>
+    <p class="contact-me__para">
+      If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+      I’d love to help with it!
+    </p>
+    <form action="https://formspree.io/f/mjvdlojw" method="POST" class="contact-me__form">
+      <input class="contact-me__form--input" type="text" name="user_name" placeholder="Lorem Ipsum">
+      <input class="contact-me__form--input" type="email" name="user_email" placeholder="loremipsum@gmail.com">
+      <textarea class="contact-me__form--textarea" type="text" name="message" id="" placeholder="Write your message here..."></textarea>
```
Comment (usorfaitheloho) [inline index.html:358]: Please add the `maxlength=30` and `maxlength=500` attribute values for username and message field respectively to enable your input have a fixed maximum length a user can input to the form. Also kindly add the required attribute to make it mandatory to add all inputs before submitting 😄

File: styles.css [hunk 2]
```diff
@@ -284,8 +284,68 @@ ul {
...
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  width: 100%;
+  height: 30vh;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+  padding: 12px 38px 12px 15px;
+  margin-bottom: 22px;
+}
```
Comment (usorfaitheloho) [inline styles.css:351]: kindly add the placeholder color given in the figma design to allow for uniformity with design given and your design and for a better UI. Also, please use the required button interaction styling for your design.

Figma design :
image

Your design:
image

button interaction :
image

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @GreetyCr  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * All linter checks passed ✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -343,5 +343,22 @@ <h2 class="about-myself__text--title">
...
+    <p class="contact-me__para">
+      If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+      I’d love to help with it!
+    </p>
+    <form action="https://formspree.io/f/mjvdlojw" method="POST" class="contact-me__form">
+      <input class="contact-me__form--input" type="text" name="user_name" maxlength="30" placeholder="Lorem Ipsum">
+      <input class="contact-me__form--input" type="email" name="user_email" maxlength="30" placeholder="loremipsum@gmail.com">
+      <textarea class="contact-me__form--textarea" type="text" name="message" id="" maxlength="500" placeholder="Write your message here..."></textarea>
+      <button class="works__button--element">Get in touch</button>
+    </form>
```
Comment (frempongdev) [inline index.html:360]: Good job adding the length validation for all your form fields. 👌🏿 . You have just a few changes to make. ♻️ 

- [ ] Please make sure you add `required` attribute to all the input fields as it is important that all fields are present before the form is submitted. ✅

**Review by frempongdev — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 🚧 

Hi @GreetyCr   👋🏼 
I'm Frempong.

You've done a great job implementing most of the requirements for this project 🚀, but there are a few changes I recommend be made. You're almost done, keep up the good work! 🤖 🎆 



 ## Highlights
✅ Great use of GitHub flow. 💯 
✅ No Linter errors. 🆗  
✅ Your project is documented professionally. 📜 
✅ Descriptive PR title, summary and commit messages. 💬 

 ## Required Changes ♻️
_Kindly Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**